<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Lungsegmentation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Scans</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>list_scans</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_scans">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="load_scan_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_scans_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="seg_menu_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Segment</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Segmentations</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>list_seg</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_seg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="save_seg_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="load_seg_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="close_seg_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="compare_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Compare</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="batch_menu_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Batch</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="slider_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="layer_slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_pic">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>list_scans</tabstop>
  <tabstop>load_scan_button</tabstop>
  <tabstop>close_scans_button</tabstop>
  <tabstop>seg_menu_button</tabstop>
  <tabstop>list_seg</tabstop>
  <tabstop>load_seg_button</tabstop>
  <tabstop>save_seg_button</tabstop>
  <tabstop>close_seg_button</tabstop>
  <tabstop>compare_button</tabstop>
  <tabstop>batch_menu_button</tabstop>
  <tabstop>layer_slider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>load_seg_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_seg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_seg_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_save_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seg_menu_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_seg_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>batch_menu_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_batch_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layer_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slider_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_scans_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close_scan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_seg_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close_seg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_scans</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>change_display()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>compare_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_compare_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_scan_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_load_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_scans</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>scan_selection_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_seg</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>seg_selection_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_seg</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>double_clicked_seg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_scans</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>double_clicked_scan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_scan()</slot>
  <slot>load_seg()</slot>
  <slot>open_save_menu()</slot>
  <slot>segment()</slot>
  <slot>open_seg_menu()</slot>
  <slot>open_batch_menu()</slot>
  <slot>compare()</slot>
  <slot>TestClick(int)</slot>
  <slot>close_scan()</slot>
  <slot>close_seg()</slot>
  <slot>show_seg()</slot>
  <slot>change_display()</slot>
  <slot>open_compare_menu()</slot>
  <slot>open_load_menu()</slot>
  <slot>scan_selection_change()</slot>
  <slot>slider_changed()</slot>
  <slot>seg_selection_changed()</slot>
  <slot>double_clicked_seg()</slot>
  <slot>double_clicked_scan()</slot>
 </slots>
</ui>
